modules/gdscript/tests/scripts/analyzer/errors/return_invalid_covariant.gd

```diff
@@ -0,0 +1,29 @@
+class BaseClass extends RefCounted:
+
+	func greetings() -> void:
+		print("Hello!")
+
+class OtherClass extends RefCounted:
+
+	func greetings() -> void:
+		print("Hello Other!")
+
+class SubClass extends BaseClass:
+	
+	func greetings() -> void:
+		print("Hello Sub Class!")
+
+class BaseGreeting extends RefCounted:
+	
+	func get_obj() -> OtherClass:
+		return BaseClass.new()
+
+class SubGreeting extends BaseGreeting:
+	
+	func get_obj() -> SubClass:
+		return SubClass.new()
+
+func test():
+	var base:BaseGreeting = SubGreeting.new()
+	var obj = base.get_obj()
+	obj.greetings()
```

Comment:
```suggestion
	obj.greetings()

```
Missing termination, same for your other `.gd` files

Reply:
Done.